wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/ConversationListTopBarViewController.swift

```diff
@@ -241,8 +241,6 @@ final class ConversationListTopBarViewController: UIViewController {
         let keyboardAvoidingViewController = KeyboardAvoidingViewController(viewController: settingsViewController)
 
         if wr_splitViewController?.layoutSize == .compact {
-            keyboardAvoidingViewController.modalPresentationStyle = .currentContext
-            keyboardAvoidingViewController.transitioningDelegate = self
```

Comment:
NOTE: this changes the way we present the settings screen to be displayed in the default modal way.

Reply:
I wonder if we should ask the design team about this change.